### ISSUE TITLE
insufficient color contrast on documentation pages

### ISSUE THREAD

**Issue by mwarner** [axios/axios#7036]
### Is your feature request related to a problem? Please describe.

In light mode, many elements have 2.95:1 contrast ratio. Dark mode is better, but still short of the mark at 3.94:1. The minimum is 4.5:1.  This negatively impacts users with even moderate visual impairment.

Please see https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html for more information.

### Describe the solution you'd like

Correct font colors to meet the 4.5:1 contrast ratio

### Additional context/Screenshots

```bash

```

**Comment by prateekbisht23** [comment]
@mwarner can you provide some screenshots or links for the reference to the issue!!

**Comment by jasonsaayman** [comment]
please could you rather open this on the docs site?

**Comment by mwarner** [comment]
@jasonsaayman Yep, will do.
